CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ Possible log types:
 ### Unreleased
 
 - [changed] Bump dependencies, including spaceapi which now uses an enum for
-  the `issue_report_channels` (#92)
+  the `issue_report_channels` and changes the in momory format of extensions
```

Comment:
```suggestion
  the `issue_report_channels` and changes the in-memory format of extensions
```

Is the in-memory format relevant at all for users of this crate? I don't think so.

Reply:
I rewrote the entry completely.